.github/workflows/release.yml

```diff
@@ -10,12 +10,16 @@ jobs:
     - name: setup go
       uses: actions/setup-go@v4
       with:
-        go-version: 1.14
+        go-version: 1.21
     - name: checkout
       uses: actions/checkout@v4
       with:
-        fetch-depth: 1
+        fetch-depth: 0
     - name: run GoReleaser
-      uses: goreleaser/goreleaser-action@v1
+      uses: goreleaser/goreleaser-action@v5
+      with:
+        distribution: goreleaser
+        version: v1.21.2
```

Comment:
I opted to pin this version since goreleaser appears to be introducing breaking changes somewhat frequently https://goreleaser.com/deprecations/